src/components/Vocal.js

```diff
@@ -116,24 +129,50 @@ const Vocal = ({
 	}
 
 	const _renderDefault = () => (
-		<div
+		<button
 			data-testid="__vocal-root__"
+			ref={buttonRef}
 			role="button"
 			aria-label={ariaLabel}
 			tabIndex={tabIndex}
-			style={className ? null : { width: 24, height: 24, cursor: !isListening ? 'pointer' : null, ...style }}
+			style={
+				className
+					? null
+					: {
+							width: 24,
+							height: 24,
+							background: 'none',
+							border: 'none',
+							padding: 0,
+							cursor: !isListening ? 'pointer' : 'default',
+							...style,
+					  }
+			}
 			className={className}
+			onFocus={_onFocus}
+			onBlur={_onBlur}
+			onClick={_onClick}
 		>
 			<Icon isActive={isListening} iconColor="#aaa" />
-		</div>
+		</button>
 	)
 
-	return (
-		SpeechRecognitionWrapper.isSupported &&
-		cloneElement(isValidElement(children) ? children : _renderDefault(), {
-			...(!isListening && { onClick: _onClick }),
-		})
-	)
+	const _renderChildren = (children) => {
+		if (SpeechRecognitionWrapper.isSupported) {
+			if (isFunc(children)) {
+				return children(startRecognition, stopRecognition)
```

Comment:
Ça pourrait valoir le coup de préciser que "stop" permet de stopper *prématurément* la reco mais qu'elle s'arrêtera d'elle-même si on ne parle pas pendant trop longtemps. Avec ce comportement, le cas d'usage du dessous n'est de toute façon pas nécessaire car la syntaxe avec une fonction couvre déjà tout ce qu'il faut.

Reply:
Ok, i'm going to add that clarification in the README. Regarding the second issue, like above, this is for compatibility purpose